Add new benchmark: Galician bench

zxcvuser opened the pull request:
GalicianBench is a benchmark for tasks in Galician that cover several evaluation areas. The datasets consist of professional translations of relevant English datasets and newly created datasets in Galician. The README.md contains detailed information on all the tasks included in the benchmark.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/EleutherAI/lm-evaluation-harness?pullRequest=2155) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/EleutherAI/lm-evaluation-harness?pullRequest=2155) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/EleutherAI/lm-evaluation-harness?pullRequest=2155) it.</sub>

zxcvuser commented on lm_eval/tasks/galician_bench/mgsm_direct_gl.yaml:
```diff
@@ -0,0 +1,27 @@
+group:
```
We will remove the group. As for translation, it is human translated. We will ask the authors to add this information in the dataset page. Thank you!

zxcvuser commented:
These are the changes done:
- Added the benchmark info in lm_eval/tasks/README.md
- Replaced "-" by "_" in the create_files script in flores_gl and added weight_by_size: false
- Run linters
- Remove grouping in mgsm, paws, xnli, and belebele tasks (they were pointing to pre-existing benchmarks)
- Changed "group" to "tag" in truthful_gl

With these, it should all be fine now. Thank you!
